cmd/grafana-agent-crow/README.md

```diff
@@ -0,0 +1,7 @@
+# Crow
+
+Crow is a tool similar to Tempo Vulture and Loki Canary that is used to smoke test Grafana Agent. Crow works by generating metrics, then validating them against Prometheus. Crow uses two endpoints; the traditional `metrics` and then `validate` that generates the results of Crow checking for successful samples. 
```

Comment:
nit:

```suggestion
Crow is a tool similar to Tempo Vulture and Loki Canary that is used to smoke test Grafana Agent. Crow works by generating metrics, then validating them against Prometheus. Crow uses two endpoints; the traditional `/metrics` and then `/validate` that generates the results of Crow checking for successful samples. 
```

Reply:
Can we also say that `/validate` should only ever be checked by an Agent that is configured to remote_write?